Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -20,6 +20,9 @@ class DefaultAudioDevice: NSObject {
     static let kAudioDeviceHeadset = "AudioSessionManagerDevice_Headset"
     static let kAudioDeviceBluetooth = "AudioSessionManagerDevice_Bluetooth"
     static let kAudioDeviceSpeaker = "AudioSessionManagerDevice_Speaker"
+    static let kStandardPlayoutDelay: UInt8 = 15
+    static let kToMicroSecond: Double = 1000000
```

Comment:
I made it a double to reduce casting, because it is always multiplicated by a double variable. Changing it to a uint32 and cast it to double every time will keep the problem. We can instead cast the double to int, but won't that also raise the same issue? And in this case, the variable is much smaller, causing an higher inaccuracy. 

Also, the result will always be divided by 1000 afterwards, already with both being an int. What do you think is the best approach?